### DISCUSSION TITLE
How does one control the log level?

### DISCUSSION THREAD

**Discussion by qwbarch** [co-log/co-log-core#12] [Q&A] [answered]
As the title mentions, how do I set the application's log level?
For example, if I wanted to hide all debug logs but still show info/warn/exceptions.
How would I be able to set this?

**Comment by alaendle** [comment, marked as answer]
@qwbarch - I think https://kowainik.github.io/posts/2018-09-25-co-log#cfilter will give you the answers you are looking for.
